cfme/tests/containers/test_pause_resume_workers.py

```diff
@@ -0,0 +1,76 @@
+from cfme.containers.provider import ContainersProvider
+from cfme.utils.appliance.implementations.ui import navigate_to
+import pytest
+
+
+pytestmark = [
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1),
+    pytest.mark.provider([ContainersProvider], scope='function')
+]
+
+
+def get_pause_resume_buttons(view):
+    # ensure resume option is exists
+    resume_option = filter(lambda item: "resume" in item.lower(), view.toolbar.configuration.items)
```

Comment:
I don't like this implementation, for a few reasons - one being lambdas should only be used when necessary, and this isn't such a case. This can easily be done without using a lambda by not using the filter method.

Second, its possible there will be multiple configuration menu items with either 'resume' or 'pause' in them, its easy to imagine MIQ features which change the items in the menu.

The specific text that's expected for a given version of cloudforms should be used within the test in order to complete the action.

If this is commonly used, and we don't want to have multiple tests defining a `cfme.utils.version.VersionPicker` object for these items, then it should be defined along with the provider class as an attribute. Or there should be a convenience method added to make the selection (like `provider.pause_worker()` and `provider.resume_worker()`

This ultimately just creates a bit of an odd pattern, and won't catch the instance where multiple items match.